docs/docs/admin/traefik-integration.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Integrating Anubis with Traefik
+---
+
+To integrate Anubis with Traefik, you need to define a new entry point in Traefik:
+
+```yml
+--entrypoints.anubis.address=:3923
+```
+
+Then, configure Anubis with the following `docker-compose.yml`. This config takes in all requests if there is no rule on port 443, and then sends the request back to Traefik this time on port 3923. Anubis preserves the `Host` header and Traefik routes again this time going to the final target webservice.
```

Comment:
`docker-compose.yml` is the old naming convention, as far as I know docker compose only still supports it for backwards compatabillity, prolly should use `compose.yml` instead

```suggestion
Then, configure Anubis with the following `compose.yml`. This config takes in all requests if there is no rule on port 443, and then sends the request back to Traefik this time on port 3923. Anubis preserves the `Host` header and Traefik routes again this time going to the final target webservice.
```

Reply:
By the way, the host is preserved but not the coming user IP.
I don’t know if there is a trick with x forward or whatever to keep the user IP.